OpenRA.Mods.Common/UpdateRules/Rules/20231010/AddSpawnActorPowerBlockedCursor.cs

```diff
@@ -0,0 +1,36 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class AddSpawnActorPowerBlockedCursor : UpdateRule
```

Comment:
doesn't this need to be added to update path?

Reply:
Added.